livecheck/special/utils.py

```diff
@@ -1,9 +1,98 @@
 from pathlib import Path
+from loguru import logger
```

Comment:
I want to get away from loguru.

```suggestion
import logging
logger = logging.getLogger(__name__)
```

Put `logger = ...` after `__all__`.